tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -136,6 +140,8 @@ def teardown_method(self):
         """
         This function will be invoked after each test function in the module.
         """
+        if self.node_master_list[0].path_exists(path=self.remote_log_dir):
```

Comment:
same comment as above? require for all tests?

Reply:
Same as above.